pkg/crc/api/client/types.go

```diff
@@ -48,6 +48,10 @@ type SetOrUnsetConfigResult struct {
 	Properties []string
 }
 
+type GetConfigRequest struct {
+	Properties []string `json:"properties"`
+}
+
```

Comment:
We have a similar struct on [L#70](https://github.com/code-ready/crc/pull/2342/commits/ec376f4faab6da4769fc1dcb5e38dbc86248c7cf#diff-b82cad2e9771588e9cabcbb993c311622f239e97b3204c1e5b2e601563177623L66)

Reply:
nvm! this is removed in later commit

Reply:
yes I didnt see it first. I simplfied it.